analyze_logs.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -ex # -e: exit early on any error. -x: write all commands to stderr
 
+date '+%Y-%m-%d'
```

Comment:
This line isn't strictly necessary any more, but I still think it's a good idea just on general principles.